apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-synthea/beneficiary_2011.csv

```diff
@@ -1,4 +1,4 @@
CD|FIPS_STATE_CNTY_AUG_CD|FIPS_STATE_CNTY_SEPT_CD|FIPS_STATE_CNTY_OCT_CD|FIPS_STATE_CNTY_NOV_CD|FIPS_STATE_CNTY_DEC_CD|V_DOD_SW|RTI_RACE_CD|MDCR_STUS_JAN_CD|MDCR_STUS_FEB_CD|MDCR_STUS_MAR_CD|MDCR_STUS_APR_CD|MDCR_STUS_MAY_CD|MDCR_STUS_JUN_CD|MDCR_STUS_JUL_CD|MDCR_STUS_AUG_CD|MDCR_STUS_SEPT_CD|MDCR_STUS_OCT_CD|MDCR_STUS_NOV_CD|MDCR_STUS_DEC_CD|PLAN_CVRG_MO_CNT|MDCR_ENTLMT_BUYIN_1_IND|MDCR_ENTLMT_BUYIN_2_IND|MDCR_ENTLMT_BUYIN_3_IND|MDCR_ENTLMT_BUYIN_4_IND|MDCR_ENTLMT_BUYIN_5_IND|MDCR_ENTLMT_BUYIN_6_IND|MDCR_ENTLMT_BUYIN_7_IND|MDCR_ENTLMT_BUYIN_8_IND|MDCR_ENTLMT_BUYIN_9_IND|MDCR_ENTLMT_BUYIN_10_IND|MDCR_ENTLMT_BUYIN_11_IND|MDCR_ENTLMT_BUYIN_12_IND|HMO_1_IND|HMO_2_IND|HMO_3_IND|HMO_4_IND|HMO_5_IND|HMO_6_IND|HMO_7_IND|HMO_8_IND|HMO_9_IND|HMO_10_IND|HMO_11_IND|HMO_12_IND|PTC_CNTRCT_JAN_ID|PTC_CNTRCT_FEB_ID|PTC_CNTRCT_MAR_ID|PTC_CNTRCT_APR_ID|PTC_CNTRCT_MAY_ID|PTC_CNTRCT_JUN_ID|PTC_CNTRCT_JUL_ID|PTC_CNTRCT_AUG_ID|PTC_CNTRCT_SEPT_ID|PTC_CNTRCT_OCT_ID|PTC_CNTRCT_NOV_ID|PTC_CNTRCT_DEC_ID|PTC_PBP_JAN_ID|PTC_PBP_FEB_ID|PTC_PBP_MAR_ID|PTC_PBP_APR_ID|PTC_PBP_MAY_ID|PTC_PBP_JUN_ID|PTC_PBP_JUL_ID|PTC_PBP_AUG_ID|PTC_PBP_SEPT_ID|PTC_PBP_OCT_ID|PTC_PBP_NOV_ID|PTC_PBP_DEC_ID|PTC_PLAN_TYPE_JAN_CD|PTC_PLAN_TYPE_FEB_CD|PTC_PLAN_TYPE_MAR_CD|PTC_PLAN_TYPE_APR_CD|PTC_PLAN_TYPE_MAY_CD|PTC_PLAN_TYPE_JUN_CD|PTC_PLAN_TYPE_JUL_CD|PTC_PLAN_TYPE_AUG_CD|PTC_PLAN_TYPE_SEPT_CD|PTC_PLAN_TYPE_OCT_CD|PTC_PLAN_TYPE_NOV_CD|PTC_PLAN_TYPE_DEC_CD|PTD_CNTRCT_JAN_ID|PTD_CNTRCT_FEB_ID|PTD_CNTRCT_MAR_ID|PTD_CNTRCT_APR_ID|PTD_CNTRCT_MAY_ID|PTD_CNTRCT_JUN_ID|PTD_CNTRCT_JUL_ID|PTD_CNTRCT_AUG_ID|PTD_CNTRCT_SEPT_ID|PTD_CNTRCT_OCT_ID|PTD_CNTRCT_NOV_ID|PTD_CNTRCT_DEC_ID|PTD_PBP_JAN_ID|PTD_PBP_FEB_ID|PTD_PBP_MAR_ID|PTD_PBP_APR_ID|PTD_PBP_MAY_ID|PTD_PBP_JUN_ID|PTD_PBP_JUL_ID|PTD_PBP_AUG_ID|PTD_PBP_SEPT_ID|PTD_PBP_OCT_ID|PTD_PBP_NOV_ID|PTD_PBP_DEC_ID|PTD_SGMT_JAN_ID|PTD_SGMT_FEB_ID|PTD_SGMT_MAR_ID|PTD_SGMT_APR_ID|PTD_SGMT_MAY_ID|PTD_SGMT_JUN_ID|PTD_SGMT_JUL_ID|PTD_SGMT_AUG_ID|PTD_SGMT_SEPT_ID|PTD_SGMT_OCT_ID|PTD_SGMT_NOV_ID|PTD_SGMT_DEC_ID|RDS_JAN_IND|RDS_FEB_IND|RDS_MAR_IND|RDS_APR_IND|RDS_MAY_IND|RDS_JUN_IND|RDS_JUL_IND|RDS_AUG_IND|RDS_SEPT_IND|RDS_OCT_IND|RDS_NOV_IND|RDS_DEC_IND|META_DUAL_ELGBL_STUS_JAN_CD|META_DUAL_ELGBL_STUS_FEB_CD|META_DUAL_ELGBL_STUS_MAR_CD|META_DUAL_ELGBL_STUS_APR_CD|META_DUAL_ELGBL_STUS_MAY_CD|META_DUAL_ELGBL_STUS_JUN_CD|META_DUAL_ELGBL_STUS_JUL_CD|META_DUAL_ELGBL_STUS_AUG_CD|META_DUAL_ELGBL_STUS_SEPT_CD|META_DUAL_ELGBL_STUS_OCT_CD|META_DUAL_ELGBL_STUS_NOV_CD|META_DUAL_ELGBL_STUS_DEC_CD|CST_SHR_GRP_JAN_CD|CST_SHR_GRP_FEB_CD|CST_SHR_GRP_MAR_CD|CST_SHR_GRP_APR_CD|CST_SHR_GRP_MAY_CD|CST_SHR_GRP_JUN_CD|CST_SHR_GRP_JUL_CD|CST_SHR_GRP_AUG_CD|CST_SHR_GRP_SEPT_CD|CST_SHR_GRP_OCT_CD|CST_SHR_GRP_NOV_CD|CST_SHR_GRP_DEC_CD|DRVD_LINE_1_ADR|DRVD_LINE_2_ADR|DRVD_LINE_3_ADR|DRVD_LINE_4_ADR|DRVD_LINE_5_ADR|DRVD_LINE_6_ADR|CITY_NAME|STATE_CD|STATE_CNTY_ZIP_CD|EFCTV_BGN_DT|EFCTV_END_DT|BENE_LINK_KEY|PTA_CVRG_STRT_DT|PTA_CVRG_END_DT|PTB_CVRG_STRT_DT|PTB_CVRG_END_DT|PTD_CVRG_STRT_DT|PTD_CVRG_END_DT
```

Comment:
Is there a test in the master branch that will get updated to check for these new fields so we can compare its results with this branch's changes?

Reply:
The new fields have been added to master and this branch has been updated with those changes so all tests are once again enabled and pass.